libs/logic-apps-shared/src/designer-client-services/lib/standard/search.ts

```diff
@@ -100,7 +100,7 @@ export class StandardSearchService extends BaseSearchService {
       const uri = `/subscriptions/${subscriptionId}/providers/Microsoft.Web/locations/${location}/apiOperations`;
       const queryParameters: QueryParameters = {
         'api-version': apiVersion,
-        $filter: `properties/trigger eq null and type eq 'Microsoft.Web/customApis/apiOperations' and ${ISE_RESOURCE_ID} eq null`,
+        $filter: `type eq 'Microsoft.Web/customApis/apiOperations' and ${ISE_RESOURCE_ID} eq null`,
```

Comment:
We need to change both the base service and standard service

Reply:
Yep the file didn't get added in the commit for some reason, fixing now